tests/core/pyspec/eth2spec/test/context.py

```diff
@@ -312,7 +312,7 @@ def wrapper(*args, **kw):
                     return None
                 run_phases = [phase]
 
-            if PHASE0 not in run_phases and ALTAIR not in run_phases:
+            if PHASE0 not in run_phases and ALTAIR not in run_phases and MERGE not in run_phases:
```

Comment:
It is checking if `run_phases` does not contain any known phase. So it would be `if not any(EXECUTABLE_PHASES.intersection(run_phases)):`

I think it's best to leave as-is. Less code, no function calls, and it is easier to reason about this change. Once we add more phases we can still change it.